superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -770,6 +784,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
   };
 
   const onChartChange = (chart: SelectValue) => {
+    getChartVisualizationType(chart);
```

Comment:
would it be easier to update the api that fetches the list of charts to include the chart type?

Reply:
That was my initial attempt, but it seems like the related objects come back only with ID and name, and it seems I would need to change FAB itself.

Reply:
ok, gotcha. sounds good.